Runtime/Systems/SkillSystems.cs

```diff
@@ -53,8 +50,7 @@ protected override void OnUpdate()
                     }
                     sbArray[i] = Skill;
                 }
-            }).WithBurst()
```

Comment:
`WithBurst` is the default, unless you're trying to be explicit here.

Reply:
Yes I try to be as explicit as possible. Like for the dependancy management.